deploy/apigee/.gitignore

```diff
@@ -0,0 +1,6 @@
+
+**/.terraform/*
+**/.terraform.lock.*
+**/terraform/*/.tmp/*
+nonprod.env
```

Comment:
Are these files persisted anywhere?

Reply:
Not sure where a good place to persist them would be. They shouldn't be public since they contain sensitive keys. Do you have ideas/what is typical?

Reply:
Check out the [gcp secrets manager](https://cloud.google.com/secret-manager/docs/creating-and-accessing-secrets). Maybe we could store prod.env and nonprod.env in datcom-apigee and datcom-apigee-dev's respective secrets managers?

Reply:
Makes sense to me! Will see if I can update the deployment script to create new secret versions automatically after deployment, and also add something to the README about pulling down the current version.